documentation/browser/plugins/rotation_from_direction.md

```diff
@@ -0,0 +1,15 @@
+#Rotation from Direction
+
+##Description
```

Comment:
seeing we have a magnitude plugin already. maybe a one line in the docs to explain this one?